website/layouts/partials/_default/banner.html

```diff
@@ -0,0 +1,16 @@
+{{- $currentVersion := index (split .Dir "/") 1 }}
+{{- $latestVersion := .Site.Params.latest }}
+{{- $notLatestVersion := ne $currentVersion $latestVersion }}
+{{- $latestUrl := replace .URL $currentVersion $latestVersion }}
+<header>
```

Comment:
Do you think it should be header? I think it's just a `div`. Also let's make it outstanding so its easy to spot. Some slightly yellow bar would do.. (:

Reply:
Done!